data/starts.txt

```diff
@@ -323,4 +323,4 @@ conversation "quickstart"
 		goto campaigns
 
 	label end
-	`The terminal aburptly flashes bright enough to make you lurch awake. On your bed rests your faithful datapad displaying the credit value on the console in your apparent dream, but also a new license. You'd best not question it; the galaxy calls.`
\ No newline at end of file
+	`The terminal aburptly flashes bright enough to make you lurch awake. On your bed rests your faithful datapad displaying the credit value on the console in your apparent dream, but also some new licenses. It would be best not question it; the galaxy calls.`
```

Comment:
I've written "a new license" to avoid discontinuities if the player chooses to start with only the default pilot's license (similar to the way the base game always refers to the player as if they're a lone crewman on a single ship).